utils.js

```diff
@@ -88,9 +88,35 @@ async function makeReactLayout() {
         'crossorigin': 'crossOrigin'
     };
 
+    const ischild = (node, tagName) => {
+        let tempNode = node;
+
+        while (tempNode != null) {
+            if (tempNode.name === tagName) {
+                return true;
+            }
+            tempNode = tempNode.parent;
+        }
+        return false;
+    };
+
+    const handleFooterNodeAndDescendant = (node, prefix) => {
```

Comment:
Can you make a new component, i'm sad how big and complicated I had to make this function.

see the `$('#creativecommons').append('<SiteVersion />');` above
Something like `$('#.license-box').append('<ReportAProblem />');` (or even after().append() type thing)

Reply:
Interesting, great suggestion. I will refactor this by creating a new component. Thanks @halkeye